tokens/Cargo.toml

```diff
@@ -10,7 +10,7 @@ homepage = "https://solana.com/"
 documentation = "https://docs.rs/solana-tokens"
 
 [dependencies]
-chrono = { version = "0.4", features = ["serde"] }
```

Comment:
this `Cargo.toml` update is unavoidable because of the need to use `.with_ymd_and_hms()`, which is added in v0.4.23.

Intentionally, I avoided to bump version in other `Cargo.toml`s to avoid downstream `chrono` update churn en masse like `solana-sdk`. As long as we don't use newer fn like `solana-tokens`. this should be safe.

that said, i think this particular `Cargo.toml` update should be harmless, considering this crate is virtually dead: https://crates.io/crates/solana-tokens/reverse_dependencies

Reply:
I think maybe I can bump the others in another PR. (I would like to make packages use the same version of deps when they are under the same workspace so I force all of them align to 0.4.22 in my workspace inheritance PR)